docs/Sprint-Review/sprint-94-summary.md

```diff
@@ -0,0 +1,80 @@
+# Sprint 94 Summary
+
+02/28/2024 - 03/12/2024
+
+Velocity (Dev): 16
+
+Velocity (UX): 2
+
+## Sprint Goal
+* Dev:
+    * Finish Cat 4 validations
+    * Validation Enhancements and Fixes
+    * Elasticsearch Improvement
+    * Kibana Group Permissions
+
+* DevOps:
+    * N/A
+
+* Design: 
+    * Error Categories GitHub Documentation
+    * Friendly name fixes (#2801)
+    * Knowledge Center Content (#2847, #2846)
+        * Submission History
+        * Error reports / data file structure
+   
+
+
+## Tickets
+### Completed/Merged
```

Comment:
@reitermb #2825 missing here.

Reply:
Adding to Ready To Merge to reflect the state as of yesterday!